test/internals/reply.test.js

```diff
@@ -544,6 +544,30 @@ test('plain string with content type application/json should be serialized as js
   })
 })
 
+test('string with content type application/json and skipSerialization() called should NOT be serialized as json', t => {
```

Comment:
Could add another test to `JSON.stringfy({ test: true })`? Just to prevent something

Reply:
What you mean? Simple send serialized via JSON.stringify data with skipSerialization()?

Reply:
Yes